tests/cleanup/remove_old_vms_aws.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Script removes virtual machines older than HOURS_LIMIT (24 hours by default) from Amazon EC2
+
+. /usr/share/beakerlib/beakerlib.sh
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        if [ -z "$AWS_ACCESS_KEY_ID" ]; then
+            rlFail "AWS_ACCESS_KEY_ID is empty!"
+        else
+            rlLogInfo "AWS_ACCESS_KEY_ID is configured"
+        fi
+
+        if [ -z "$AWS_SECRET_ACCESS_KEY" ]; then
+            rlFail "AWS_SECRET_ACCESS_KEY is empty!"
+        else
+            rlLogInfo "AWS_SECRET_ACCESS_KEY is configured"
+        fi
+
+        # VMs older than HOURS_LIMIT will be deleted
+        HOURS_LIMIT="${HOURS_LIMIT:-24}"
+        export TIMESTAMP=`date -u -d "$HOURS_LIMIT hours ago" '+%FT%T'`
+        export AWS_REGION="${AWS_REGION:-us-east-1}"
+
+        rlLogInfo "HOURS_LIMIT=$HOURS_LIMIT"
+        rlLogInfo "AWS_REGION=$AWS_REGION"
+
+        for package in ansible python3-boto3; do
```

Comment:
You can also do `pip install ansible[amazon]` to fetch all dependencies. 

@bcl given that we trash the system running this script afterwards which one do you prefer: pip or dnf ?

Reply:
dnf. reduces the chances for differences resulting in weird problems.